src/strings.rs

```diff
@@ -20,4 +20,9 @@ pub const MISSING_PLAY_QUERY: &str =
     "Missing query for this command! Either add a URL or keywords.";
 
 pub const MISSING_INDEX_QUEUE: &str = "Missing an index!";
+
 pub const NO_SONG_ON_INDEX: &str = "There is no queued song on that index!";
+
+pub const PREFIX_UPDATED: &str = "Updated prefix!";
+
+pub const MISSING_PREFIX: &str = "⚠️ Please send a character as an argument.\nExample: `!prefix ~`";
```

Comment:
Nice unicode emoji. @aquelemiguel  would be proud.

Reply:
I think we should have more actually :eyes: